.github/workflows/build.yml

```diff
@@ -131,33 +98,136 @@ jobs:
           path: |
             /out/drakvuf-bundle*.deb 
             /out/xen-hypervisor*.deb
-          key: drakvuf-bundle-${{ env.DRAKVUF_COMMIT }}
+            /opt/xtf/tests/example/test-hvm64-example
+          key: drakvuf-bundle-${{ env.DRAKVUF_COMMIT }}-${{ matrix.distro }}-${{ matrix.version }}
       - if: ${{ steps.cache-drakvuf-bundle.outputs.cache-hit != 'true' }}
         name: Build Drakvuf bundle
         run: |
           cp -ra . /build
           cd /build
           sh drakvuf/package/depends.sh
-          bash ci/build_bundle.sh "debian:buster"
+          bash ci/build_bundle.sh "${{ matrix.distro }}:${{matrix.version}}"
+      - name: Install Drakvuf bundle
+        run: |
+          export DEBIAN_FRONTEND=noninteractive
+          apt install -y /out/drakvuf-bundle*.deb
+      - if: ${{ steps.cache-drakvuf-bundle.outputs.cache-hit != 'true' }}
+        name: Build Xen Test Framework
+        working-directory: /opt
+        run: |
+          git clone https://xenbits.xen.org/git-http/xtf.git
+          cd xtf
+          git checkout bf1c4eb6cb52785cf539eb83752dfcecfe66c5d1
+          make -j4
+      - name: Build draksetup tools
+        run: |
+          apt install -y make pkg-config gcc libglib2.0-dev
+          make -C ./drakrun/drakrun/tools
+          cp /opt/xtf/tests/example/test-hvm64-example ./drakrun/drakrun/tools/
       - uses: actions/upload-artifact@v3
         with:
-          name: drakvuf-bundle-debs
+          name: drakvuf-bundle-debs-${{ matrix.distro }}-${{ matrix.version }}
           path: |
             /out/drakvuf-bundle*.deb 
             /out/xen-hypervisor*.deb
+      - uses: actions/upload-artifact@v3
+        with:
+          name: draksetup-tools-${{ matrix.distro }}-${{ matrix.version }}
+          path: |
+            drakrun/drakrun/tools/*
+  package_drakcore:
+    needs: [lint_drakcore, lint_drakcore_frontend]
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - distro: debian
+            version: buster
+          - distro: debian
+            version: bullseye
+    runs-on: ubuntu-latest
+    container: "${{ matrix.distro }}:${{matrix.version}}"
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build drakcore
+        run: |
+          export DEBIAN_FRONTEND=noninteractive
+          apt-get update
+          apt-get install -y wget curl python3 python3-venv python3-pip dh-virtualenv debhelper devscripts
+          curl -sL https://deb.nodesource.com/setup_14.x | bash -
+          apt-get install -y nodejs
+          cd drakcore
+          package/find-python.sh
+          dpkg-buildpackage -us -uc -b
```

Comment:
TODO: This part can be actually moved to composite action. Will be done in another PR.